.github/scripts/support/labels.js

```diff
@@ -0,0 +1,41 @@
+const labels = {
```

Comment:
Nitpick: should we call this file `resources.js` so we can add additional objects in here like we do with the components?

Reply:
Good idea. Will rename!